app/utilities/schema.py

```diff
@@ -16,38 +17,59 @@
 logger = get_logger()
 
 SCHEMA_DIR = "schemas"
-TEST_SCHEMA_DIR = "test_schemas"
+TEST_SCHEMA_DIR = "schemas/test"
 LANGUAGE_CODES = ("en", "cy", "ga")
 
 LANGUAGES_MAP = {"test_language": [["en", "cy"], ["en", "ga"]]}
 
 
 @lru_cache(maxsize=None)
-def get_schema_list(language_code: str = DEFAULT_LANGUAGE_CODE) -> List:
-    return list(get_schema_path_map(include_test_schemas=True)[language_code].keys())
+def get_schema_list(language_code: str = DEFAULT_LANGUAGE_CODE) -> dict[str, list]:
+    return {
+        survey_type: list(schemas_by_language[language_code])
+        for survey_type, schemas_by_language in get_schema_path_map(
+            include_test_schemas=True
+        ).items()
+        for lang in schemas_by_language
+        if lang == language_code
+    }
 
 
 @lru_cache(maxsize=None)
-def get_schema_path_map(include_test_schemas: Optional[bool] = False) -> Mapping:
-    dirs = [SCHEMA_DIR]
-    if include_test_schemas:
-        dirs.append(TEST_SCHEMA_DIR)
+def get_schema_path(language_code, schema_name):
+    for schemas_by_language in get_schema_path_map(include_test_schemas=True).values():
+        schema_path = schemas_by_language.get(language_code, {}).get(schema_name)
+        if schema_path:
+            return schema_path
 
-    return {
-        language_code: {
-            Path(schema_file).with_suffix("").name: schema_file
-            for schema_dir in dirs
-            for schema_file in glob(f"{schema_dir}/{language_code}/*.json")
+
+@lru_cache(maxsize=None)
+def get_schema_path_map(include_test_schemas: Optional[bool] = False) -> Mapping:
+    schemas = {}
+    for survey_type in os.listdir(f"{os.getcwd()}/{SCHEMA_DIR}"):
+        if not include_test_schemas and survey_type == "test":
+            continue
+
+        schemas[survey_type] = {
+            language_code: {
+                Path(schema_file).with_suffix("").name: schema_file
+                for schema_file in glob(
+                    f"{os.getcwd()}/{SCHEMA_DIR}/{survey_type}/{language_code}/*.json"
```

Comment:
The `getcwd` can be removed since this will be run from project root.
```suggestion
    for survey_type in os.listdir(SCHEMA_DIR):
        if not include_test_schemas and survey_type == "test":
            continue

        schemas[survey_type] = {
            language_code: {
                Path(schema_file).with_suffix("").name: schema_file
                for schema_file in glob(
                    f"{SCHEMA_DIR}/{survey_type}/{language_code}/*.json"
```